source/endpoints/keywords.md.erb

```diff
@@ -0,0 +1,167 @@
+      "best_position":1,
+      "position_changed_during_last_day":0,
+      "position_changed_during_last_week":0,
+      "position_changed_during_last_month":0,
+      "historic_positions":[6,6,6,6,6,6,6,6,6,6,5,5,5,5,8,8,8,8,8,8,8,8,8,7,8,7,7,6,6,6,6,5],
+      "last_historic_position_on":"2019-06-04",
+      "position_change":0,
+      "available_serp_features":{"places_image":false,"organic":true,"local_pack":false,"knowledge_panel":true,"featured_snippet":false}
+    }
+  ]
+```
+
+### HTTP Request
+
+`GET https://api.nightwatch.io/api/v1/urls/URL_ID/keywords?access_token=ACCESS_TOKEN`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+sort      | The name of the field to sort by. <br><br>Possible values: id, query, google_gl, google_hl, last_processed_at, created_at, updated_at, position, position_type, position_organic, position_places, position_places_image, position_changed_by, position_changed_status, position_changed_at, position_local_pack, position_featured_snippet, position_knowledge_panel, local_search, position_changed_during_last_day, position_changed_during_last_week, position_changed_during_last_month, mobile, engine, previous_position, best_position, available_serp_features, adwords_global_search_volume, adwords_global_average_cpc, adwords_local_search_volume, adwords_local_average_cpc.
+direction | The direction to sort by. <br><br>Possible values: asc, desc.
+search    | A general search string to filter by keywords' tags, result URL, location.
+
+## Add keywords
+
+```shell
+  curl 'https://api.nightwatch.io/api/v1/urls/URL_ID/keywords/batch_create?access_token=ACCESS_TOKEN' \\
+    -H 'Content-Type: application/json' \\
+    --data-binary '{"keywords":"keyword1\nkeyword2\nkeyword3","tags":[],"google_gl":"us","google_hl":"en","mobile":false,"desktop":true,"search_engine":"google"}' \\
+    --compressed
+```
+
+> The above command returns JSON structured like this:
+
+```json
+  [
+    { "id":1,
+      "query":"keyword1",
+      ...
+    }, {
+      "id":2,
+      "query":"keyword2",
+      ...
+    },
+    ...
+  ]
+```
+
+### HTTP Request
+
+`POST https://api.nightwatch.io/api/v1/urls/URL_ID/keywords?access_token=ACCESS_TOKEN`
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+keywords*  | Search queries separated by new lines (\n)
+google_hl* | Search language code (see <a href="https://developers.google.com/custom-search/docs/ref_languages">here</a> for available options)
+google_gl*    | Search geolocation code (see <a href="https://developers.google.com/custom-search/docs/xml_results_appendices#countryCodes">here</a> for available options)
+search_engine* | Search engine ('google', 'youtube', 'places', 'bing' or 'yahoo')
+adwords_location_id | Search location Criteria ID (see <a href="https://developers.google.com/ad-exchange/rtb/geotargeting">here</a> for available options)
+mobile | Search adjusted for mobile devices (true or false, default: true)
```

Comment:
Is `true` really the default?

Reply:
Of course not. Good catch. I carelessly copied this from RankTrackr's docs. Which are wrong. Added a ticket to fix it there too.